src/console_cmds.cpp

```diff
@@ -1049,6 +1049,52 @@ DEF_CONSOLE_CMD(ConNewGame)
 	return true;
 }
 
+DEF_CONSOLE_CMD(ConNewHeightMapGame)
+{
+	bool heightmap_loaded = true;
+
+	if (argc == 0) {
+		IConsoleHelp("Load a game by name or index. Usage: 'newheightmapgame [<file | number> [seed]]'");
+		IConsoleHelp("The server can force a new game using 'newheightmapgame'; any client joined will rejoin after the server is done generating the new game.");
+	}
+
+	if (argc >=1 && argc <= 3) {
+		const char *file = NULL;
+
+		if (argc == 1) {
+			file = _file_to_saveload.title;
+		}
+
+		if (argc >= 2) {
+			file = argv[1];
+		}
+
+//		_console_file_list.ValidateFileList();
```

Comment:
This commented-out line needs to go. Or should it not be commented-out?

Reply:
Nah it shall disappear, as the ValidateFileList, despite its name, is oriented towards a specific filetype, as in normal savegames.
I wanted to pinpoint the oddity in this function behaviour.

That's why I have to manually invoke `BuildFileList(FT_HEIGHTMAP, SLO_LOAD);`

Reply:
Maybe it warrants a comment in that direction?

Reply:
Wilco.